install/migrations/update_0.84.0_to_0.84.1.php

```diff
@@ -72,7 +72,7 @@ function update0840to0841() {
                             'glpi_notificationtemplatetranslations' => 'content_text'];
    foreach ($fields_to_clean as $table => $field) {
       foreach ($DB->request($table) as $data) {
-         $text  = Toolbox::unclean_html_cross_side_scripting_deep($data[$field]);
+         $text  = @Toolbox::unclean_html_cross_side_scripting_deep($data[$field]);
```

Comment:
I do not think this is a good idea to mute errorrs; this one will have to be rewritten aswell, doesn't it?

Reply:
Maybe this whole loop can be dropped, but I have to make manual tests on this.

This has been done to fix https://forum.glpi-project.org/viewtopic.php?id=32196 in commit f19be70f24d0686a4477923fed0b6fc60782f901 .

If I manually put a value corresponding to thoose provided in forum thread, it is correctly displayed without having to run this migration.

Reply:
I made some tests and I think we could drop this part (done in my last commit).

Indeed, the only purpose was to re-encode HTML entities from a numeric encoding to the default encoding (e.g. from `&#233;` to `&eacute;`). As both encoding are valid HTML encodings, I do not get why it has been done like this.